openrpc.json

```diff
@@ -343,46 +343,97 @@
         }
       ],
       "summary": "Tracks a token in MetaMask.",
-      "description": "Requests that the user track the specified token in MetaMask. Returns a boolean indicating if the token was successfully added. Once added, the token is indistinguishable from those added using legacy methods, such as a centralized registry. Originally introduced by [EIP-747](https://eips.ethereum.org/EIPS/eip-747), however the current specification for EIP-747 has diverged from the implementation in MetaMask. The implementation in MetaMask, described below, matches the version of EIP-747 found here: https://github.com/ethereum/EIPs/blob/36102e0c82630e2654fd019f54914330d1c37f74/EIPS/eip-747.md",
+      "description": "Requests that the user track the specified token in MetaMask. Returns a boolean indicating if the token was successfully added. Once added, the token is indistinguishable from those added using legacy methods, such as a centralized registry. Introduced by [EIP-747](https://eips.ethereum.org/EIPS/eip-747).",
       "paramStructure": "by-name",
       "params": [
         {
           "name": "type",
           "schema": {
             "type": "string",
-            "description": "Currently only supports ERC-20 tokens.",
-            "enum": [
-              "ERC20"
-            ]
-          }
+            "description": "Supports ERC-20, ERC-721, and ERC-1155 tokens.",
```

Comment:
I feel like we don't have a great handle across all our codebases on the use of the term token vs asset. Part of me feels like asset is less confusing when we're referring to both ERC20 tokens and ERC1155/ERC721 tokens since the latter are much more commonly referred to as NFTs and, as such, thought of as a separate category from (fungible) tokens. Anyone have thoughts on this and whether it makes sense to change the term `token` to `asset` here? cc @BelfordZ @vandan @shanejonas @jiexi

Reply:
Token is in the name for these types so we could stick with the term even though our method name uses the more generic watch**Asset** ¯\_(ツ)_/¯

Reply:
Can we update the method-level and type-level description to specify that ERC-721 & ERC-1155 support is considered stability level (1) "Experimental" and subject to change?